### DISCUSSION TITLE
How do I enable DEBUG in the browser

### DISCUSSION THREAD

**Discussion by jeffsee55** [micromark/micromark#134] [Q&A] [answered]
I'm not sure how to apply [these instructions](https://github.com/micromark/micromark#size--debug) when I'm working with browser-only code. I've tried setting `localStorage.debug="micromark"` which was kind of mentioned [here](https://github.com/micromark/micromark/discussions/102#discussioncomment-1761245) as well.

**Comment by wooorm** [comment, marked as answer]
Sort of like that example shows!

a) bundle with the `development` condition on
b) configure your browser: https://github.com/debug-js/debug#browser-support

**Reply by jeffsee55** [reply]
Thanks!
